cypress/integration/login.js

```diff
@@ -21,4 +21,11 @@ describe('Login functionality', function() {
         cy.get('.user-info__title').contains('TestCon User 9')
         cy.get('.main-nav').find('li').should('have.length', 2)
     })
+
+    it('Validate what date is selected as today', function() {
+
+        cy.get('.calendar--selected').contains('13')
```

Comment:
Will it work tomorrow? Does this selector only finds one element?

Reply:
Thank you. Have made the changes